lib/mi_bridges/driver/cash_on_hand_page.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class CashOnHandPage < BasePage
```

Comment:
for the other pages like this I've been naming them`MoreAbout XUZ` to try and be as consistent with the title as possible. Thoughts on doing that here too? Or should we rename all?